grafana/grafana
# Graphite Plugin: Remove angular dependencies for graphite annotations

Fixes https://github.com/grafana/grafana/issues/50458

**What this PR does / why we need it**: This PR removes angular dependencies for graphite, creates annotations with a react component, and migrates the legacy annotations.

**Special notes for your reviewer**:
To test this PR, do the following:

1. Check out main.
2. Create a dashboard with a graphite datasource
3. Create an annotation with a tag or a query.
4. Save the Dashboard.
5. Check that the annotation is displayed on the dashboard.
6. Checkout this branch.
7. Check that the previous annotation is still displayed.
8. Create a new annotation and check that it is displayed.
